bootleg/utils/classes/nested_vocab_tries.py

```diff
@@ -270,7 +271,7 @@ def build_trie(self, input_dict: Dict[str, Any], max_value: int):
         trie = marisa_trie.RecordTrie(
             self._get_fmt_string(max_value), zip(all_keys, all_values)
         )
-        print(
+        logging.debug(
```

Comment:
So figured out a small issue with this. Because we just import logging witout making a logger, all of the number debug logs are printed as well. The solution is to instatiate a logger at the very top before `flatten()`.

`logger = logging.getLogger(__name__)`

Then call logger.debug...

Reply:
done